src/index.js

```diff
@@ -67,14 +67,58 @@ AFRAME.registerComponent('streetmix-loader', {
     streetmixStreetURL: { type: 'string' },
     streetmixAPIURL: { type: 'string' },
     showBuildings: { default: true },
-    name: { default: '' }
+    name: { default: '' },
+    streetmixJSON: { type: 'string', default: '' }
```

Comment:
I think this could be set directly on the `street` component instead of `streetmix-loader` component? there is already a property called `JSON` on the `street` component?